snapcraft.yaml

```diff
@@ -2,7 +2,7 @@ name: nextcloud
 version: 11.0.1snap3
 summary: Nextcloud Server
 description: |
- Access, share and protect your files, calendars, contacts, communication and
+ Nextcloud server running on Apache with MySQL. Access, share, and protect your files, calendars, contacts, communications, and
  more at home and in your enterprise.
```

Comment:
Sorry for the delay on this. Would it work to say "Access, share and protect your files, calendars, contacts, communication and more at home and in your enterprise with the Nextcloud Server." ?